### PR TITLE
KiYong FrontendMentor Assignment (Stats Card Component)

### PR DESCRIPTION
**Pull request by kiyonggg**

### CODE REVIEW

File: src/pages/FrontendMentor/KYStatsCard/styles/KYStatsCard.styled.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    text-align: center;
+    padding: 20px;
+  }
+`
+export const ImageContainer = styled.div`
+  width: 50%;
+  background-color: hsl(277, 64%, 61%);
+  @media (max-width: 375px) {
+    width: 100%;
+  }
```
Comment (xinyee20) [inline src/pages/FrontendMentor/KYStatsCard/styles/KYStatsCard.styled.ts:54]: Add 10px to border-bottom-right-radius and border-top-right-radius to make the image side border rounded

**Review by xinyee20 — CHANGES_REQUESTED** [review]
The header overflows out of the InformationCard in both desktop/mobile view. One quick-fix approach is to add overflow:auto, but the text will be scrollable instead of the page, another is to adjust the width proportion, the font-size of the headers, the margin of the InformationContainer, add change to min-height instead of height to allow the height of the card to adjust to the text and make the screen scrollable vertically if height goes beyond viewport etc. Can consider adding screenshot of how the implementation looks like on your side for reviewers to have a better idea of how you intended it to look.

File: src/pages/FrontendMentor/KYStatsCard/styles/KYStatsCard.styled.ts (hunk 1 above)
Comment (kiyonggg) [reply src/pages/FrontendMentor/KYStatsCard/styles/KYStatsCard.styled.ts:54]: resolved in commit f82d812

**Review by xinyee20 — APPROVED** [review]
lgtm

**Pull request merged**